README.md

```diff
@@ -32,15 +32,40 @@ else:
 **Node:**
 
 ```node
-var api = require('api').config('API_KEY');
+let api = require('api').config('API_KEY');
 
 api('math').run('multiply', {
     'x': 1,
     'y': 2,
 }, function(err, res) {
   if(err) return console.log(err);
   console.log(res);
-})
+});
+```
+# Additional SDK Funcationality
+
+##  Files
+
+Build services support invoking with files as data. Invoking with a stream, buffer, or parameter to `api.file` (detailed below) should be supported. Responses containing buffers might also need to be parsed to the appropriate data type for the specific language.
+
+### api.file
+
+`api.file` should be able to take a local path to a file or a url, and convert it to a format that can be sent to build (stream or buffer).
+
+Example (Node):
+
+```node
+let api = require('api').config('API_KEY');
+
+api('file-test').run('scale', { image: api.file('https://files.readme.io/88d46f2-small-logo.png'), width: 1000 }, (err, response) => {
```

Comment:
Does this API exist?

Reply:
It's one I have locally, I can deploy it to prod once we merge all the file stuff